pkgs/development/libraries/accounts-qt/default.nix

```diff
@@ -11,8 +11,29 @@ mkDerivation rec {
     owner = "accounts-sso";
   };
 
+  # Backport Qt 6 support
+  # Upstream: https://gitlab.com/accounts-sso/libaccounts-qt/-/merge_requests/21
+  patches = [
+    (fetchpatch {
+      url = "https://gitlab.com/accounts-sso/libaccounts-qt/-/commit/29fd38e02920b255f7ef080609e191ce0f1afbfc.diff";
+      hash = "sha256-0yTHea/MT8rXjudOEjMwlDXZTPSPsg+SXwNWNQUw6Fs=";
+    })
+    (fetchpatch {
+      url = "https://gitlab.com/accounts-sso/libaccounts-qt/-/commit/c570c96c063a0f873f08f21cc2ebd1b7adba5134.diff";
+      hash = "sha256-cJaAIDYdSgATeCyVHVdHWlzW0hpfbXL3WK3B8C6nBGQ=";
+    })
+    (fetchpatch {
+      url = "https://gitlab.com/accounts-sso/libaccounts-qt/-/commit/102cf562f9d143de2f93ba40f8448c1d531c2c7b.diff";
+      hash = "sha256-AFmzEdlAkcy4Zg/YwHpkGvAGMCODIgAEhEy1ATfpkdM=";
+    })
+    (fetchpatch {
+      url = "https://gitlab.com/accounts-sso/libaccounts-qt/-/commit/3107822e036300684c2bdf587838a110eea8ba30.diff";
+      hash = "sha256-sYZolEvv5J8eHq2CPEd3e9s0V9MZ6ViW7C1Z2vMxjZ0=";
+    })
+  ];
+
   propagatedBuildInputs = [ glib libaccounts-glib ];
-  nativeBuildInputs = [ doxygen pkg-config qmake ];
+  nativeBuildInputs = [ doxygen pkg-config qmake wrapQtAppsHook ];
 
   # remove forbidden references to $TMPDIR
   preFixup = ''
```

Comment:
the website should point to https://gitlab.com/accounts-sso/libaccounts-qt

Reply:
We'll see where this ends up.